components/compositing/webview.rs

```diff
@@ -73,9 +78,14 @@ impl Drop for WebView {
 }
 
 impl WebView {
-    pub(crate) fn new(id: WebViewId, rect: DeviceRect, global: Rc<RefCell<ServoRenderer>>) -> Self {
+    pub(crate) fn new(
+        renderer_webview: Box<dyn RendererWebView>,
+        rect: DeviceRect,
+        global: Rc<RefCell<ServoRenderer>>,
+    ) -> Self {
         Self {
-            id,
+            id: renderer_webview.id(),
```

Comment:
Maybe the `id` field could be removed and consumers can be switched to `renderer_webview.id()`?

Reply:
I did think about this, but it's accessed so many places, it made sense to me to cache it.